proto/config.go

```diff
@@ -31,6 +32,12 @@ import (
 // NodeID is a custom type for a cockroach node ID. (not a raft node ID)
 type NodeID int32
 
+// String implements the fmt.Stringer interface.
+// It is used to format the ID for use in Gossip keys.
+func (n NodeID) String() string {
```

Comment:
I'm confused - do you expect problems with something like this? http://play.golang.org/p/4cj8PIaNC7